share/picongpu/tests/KHI_growthRate/include/picongpu/param/particle.param

```diff
@@ -27,7 +27,7 @@
 #include <pmacc/math/operation.hpp>
 
 #ifndef PARAM_GAMMA
-#    define PARAM_GAMMA 1.021
+#    define PARAM_GAMMA 1.2
```

Comment:
please discuss with @PrometheusPi, this can be set via cmakeFlags provided in the PR #4356

Reply:
Without a working PICMI this will be difficult to change. In the example, I could define the line directly and not introduce an additional variable.

Reply:
I think what @psychocoderHPC want to say is that PARAM_GAMMA need not be a compile time setting but rather only runtime.
And yes we have a working PICMI for the newest interface once #4364 is merged and we can talk about a PCIMI integration, should be straightforward.

Reply:
@MSVoss 

You can change setups by adding cases into the file `cmakeFlags` see the original KHI for an example https://github.com/ComputationalRadiationPhysics/picongpu/blob/debc2f5a3646498a99379a91a9bc8822de3c420a/share/picongpu/examples/KelvinHelmholtz/cmakeFlags#L32-L34

By default case zero is build. If you like to compile a different case you can run `pic-build -t 1` to build the case one.